pkg/daemon/cleanup/cleanup.go

```diff
@@ -194,6 +196,33 @@ func removeServices(svcMgr manager.Manager, services []servicescm.Service) error
 			servicesRemoved = append(servicesRemoved, service.Name)
 		}
 	}
+
+	if bestEffort {
```

Comment:
should this happend only if `len(failedRemovals) > 0` ?

Reply:
Nope those failures would be from the services CM, while the best effort removals are a separate concept: any services NOT in the services CM anymore but still running